nautobot/extras/datasources/git.py

```diff
@@ -224,126 +224,127 @@ def refresh_git_config_contexts(repository_record, job_result, delete=False):
 def update_git_config_contexts(repository_record, job_result):
     """Refresh any config contexts provided by this Git repository."""
     config_context_path = os.path.join(repository_record.filesystem_path, "config_contexts")
-    if not os.path.isdir(config_context_path):
-        return
-
     managed_config_contexts = set()
     managed_local_config_contexts = defaultdict(set)
 
-    # First, handle the "flat file" case - data files in the root config_context_path,
```

Comment:
The diffs here are messy but it's mostly just an indentation change, from:

```
if not os.path.isdir(config_context_path):
    return
# add/update contexts
# remove stale contexts - whoops, we need to do this even if it isn't a directory!
```

to

```
if os.path.isdir(config_context_path):
    # add/update contexts
# remove stale contexts regardless
```